README.md

```diff
@@ -1,4 +1,56 @@
 # Bootstrap Playground
 
-## Getting started
+## Host the Webpage locally
```

Comment:
Let's rather create an `examples` folder, than using multiple repositories. In `index.html`, where it says "Nothing here yet...", We can link an example page placed at `examples/example-xxxx.html`